README.md

```diff
@@ -1,27 +1,71 @@
-# Colours [![Build Status](https://circleci.com/gh/Financial-Times/o-colors.png?style=shield&circle-token=0d3884dacba2fdd8aec8afbd2eaa84c79040ddd5)](https://circleci.com/gh/Financial-Times/o-colors)
+o-colors [![Build Status](https://circleci.com/gh/Financial-Times/o-colors.png?style=shield&circle-token=0d3884dacba2fdd8aec8afbd2eaa84c79040ddd5)](https://circleci.com/gh/Financial-Times/o-colors)
+
+The color palette for the FT masterbrand and sub-brand products.
+
+- [Usage](#usage)
+	- [Sass](#sass)
+	- [Markup](#markup)
+	- [CSS Variables](#css-variables)
+- [Migration guide](#migration-guide)
+	- [Upgrading from v3.x.x to v4.x.x](#upgrading-from-v3.x.x-to-v4.x.x)
+- [Contact](#contact)
+- [Licence](#licence)
 
-This is an [Origami](http://origami.ft.com/) module that provides variables defining the FT digital colour palette, and helpers to use them in your products.  For installation instructions, see the [registry page](http://registry.origami.ft.com/components/o-colors).
 
 ## Usage
 
-There are a number of ways of using colours in your component or product.  Here they are from most to least preferred.  All mixin and function approaches require you to import the module into your own build, while the final method of using predefined classes is compatible with using the build service.
+There are a number of ways of using colors in your component or product. o-colors can be used via the [Build Service](https://www.ft.com/__origami/service/build/v2/), but it is recommended you import the Sass into your project to make use of the many Sass mixins and functions available.
+
+
+### Sass:
+
+As with all Origami components, o-colors has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than incorporating its mixins into your own Sass) set `$o-colors-is-silent : false;` in your Sass before you import the o-colors Sass:
+
+```scss
+$o-colors-is-silent: false;
+@import 'o-colors/main';
+```
+
+#### Colours and accessibility
+
+o-colors has been built to help bridge the gap between design and development by providing functionality to help create colours dynamically from a central palette as well as generate contrasted text colours based on an elements background color.
+
+You can create tints of a color with the [oColorsGetTint](#tint-palette-colors) function. The function takes a palette color name and a brightness value (based on HSB color) to return a tint of the palette color.
```

Comment:
Would maybe backtick these so that they appear clearly as method names when scanning